websites/F/Foxford/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "cramatsu",
+		"id": "226622016986415104"
+	},
+	"service": "Foxford",
+	"description": {
+		"en": "Foxford is an online school for students in grades 1-11, teachers and parents.",
+		"ru": "Фоксфорд — онлайн-школа для учеников 1-11 классов, учителей и родителей."
+	},
+	"url": "foxford.ru",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/41WDw6H.jpeg",
+	"thumbnail": "https://i.imgur.com/41WDw6H.jpeg",
```

Comment:
The thumbnail **should** preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is **not** available.

Reply:
https://imgur.com/ZPIEalu
Is it ok?

Reply:
And another one that was made by me
https://imgur.com/5DZ70gr

Reply:
use [imgur.com/ZEnNmek](https://i.imgur.com/ZEnNmek.png)